### ISSUE TITLE
How to open download window？

### ISSUE THREAD

**Issue by shjanken** [mooz/keysnail#219]
in vimperator . I can use `:donwload` to open download window.
How to do same thing in `keysnail` without mouse press?

thank you very much

**Comment by wisd0me** [comment]
I'm interested in that too, didn't found a solution in existing functions

**Comment by ncaq** [comment]
KeySnail does not matter.
You can open it with firefox's native key shortcut.
In my environment `C-Y`.
<https://support.mozilla.org/ja/kb/keyboard-shortcuts-perform-firefox-tasks-quickly>

**Comment by wisd0me** [comment]
Nah, keysnail still matters. I am aware that by default it's Ctrl+Shift+Y, and I want to rebind it with keysnail. Still looking for solution.
